### PR TITLE
Update redirect

### PR DESCRIPTION
**Pull request by quibusus**

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +1 @@
-nodeschool.cz
+nodeschool.cz
```
Comment (bajtos) [inline CNAME]: This change seems unrelated to me, can you revert it please?

**Review by bajtos — CHANGES_REQUESTED** [review]

File: CNAME (hunk 1 above)
Comment (quibusus) [reply CNAME]: reverted

**Review by bajtos — APPROVED** [review]

**Comment by bajtos** [comment]
Landed, thank you for the contribution!
